### PR TITLE
Release notes for v0.6.0

### PR DESCRIPTION
**Pull request by fitzthum**
lmk if I forgot anything

### CODE REVIEW

File: releases/v0.6.0.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Support for attesting pod VMs with Azure vTPMs on SEV-SNP
+- Support for using Project Amber as an attestation service
+- Support for Cosign signature validation with s390x
+- Pulling guest images with many layers can no longer cause guest CPU starvation.
+- Attestation Service upgraded to avoid several security issues in Go packages.
+- Several CoCo components are now also shipped in the Kata Containers payload bundle.
```
Comment (fidencio) [inline releases/v0.6.0.md:17]: I'd drop this part.

There are no "CoCo components" being shipped in the Kata Containers payload.

**Review by sameo — APPROVED** [review]

File: releases/v0.6.0.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  * The [oci-crypt](https://github.com/containers/ocicrypt) container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
+  * Image repository support for encrypted images is unequal
+- CoCo currently requires a custom build of `containerd`
+  * The CoCo operator will deploy the correct version of `containerd` for you
+  * Changes are required to delegate `PullImage` to the agent in the virtual machine
+  * The required changes are not part of the vanilla `containerd`
+  * The final form of the required changes in `containerd` is expected to be different
+  * `crio` is not supported
```
Comment (bpradipt) [inline releases/v0.6.0.md:49]:
```suggestion
`crio` is supported only with `cloud-api-adaptor`

File: releases/v0.6.0.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- CoCo is not fully integrated with the orchestration ecosystem (Kubernetes, OpenShift)
+  * OpenShift support is not yet complete.
+  * Existing APIs do not fully support the CoCo security and threat model. [More info](https://github.com/confidential-containers/community/issues/53)
+  * Some commands accessing confidential data, such as `kubectl exec`, may either fail to work, or incorrectly expose information to the host
+  * Container image sharing is not possible in this release
+  * Container images are downloaded by the guest (with encryption), not by the host
+  * As a result, the same image will be downloaded separately by every pod using it, not shared between pods on the same host. [More info](https://github.com/confidential-containers/community/issues/66)
```
Comment (bpradipt) [inline releases/v0.6.0.md:56]:
```suggestion
- CoCo integration with Kubernetes is work in progress
  * Current APIs do not fully support the CoCo security and threat model. [More info](https://github.com/confidential-containers/community/issues/53)
  * Some commands such as `kubectl exec`, might incorrectly expose confidential information to the host
  * Container image sharing between Pods is not possible in this release
  * Container images are downloaded inside the guest (with encryption), and as a result, the same image will be downloaded separately for every pod using it, not shared between pods running on the same host. [More info](https://github.com/confidential-containers/community/issues/66)
```

Comment (bpradipt) [reply releases/v0.6.0.md:56]: @fitzthum I felt the limitations can be reworded in a positive way and hence suggested the above rewrite :-). 
It's not must and feel free to ignore the suggestion.

Comment (magowan) [reply releases/v0.6.0.md:56]: I like @bpradipt proposed changes , all for more positive :-)

File: releases/v0.6.0.md (hunk 2 above)
Comment (magowan) [reply releases/v0.6.0.md:49]: Yes would be a fairer way to relate to crio.

**Review by magowan — CHANGES_REQUESTED** [review]

**Comment by magowan** [comment]
Think we should update the crio sentence, regarding the download images comment I prefer @bpradipt wording and that could still include the link to the issue but will go with what others think on that.

**Review by ariel-adam — COMMENTED** [review]
I'd remove the dot at the end of the 2 last bullet points to be consistent with the previous 3 points

File: releases/v0.6.0.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Support for attesting pod VMs with Azure vTPMs on SEV-SNP
+- Support for using Project Amber as an attestation service
+- Support for Cosign signature validation with s390x
+- Pulling guest images with many layers can no longer cause guest CPU starvation.
```
Comment (ariel-adam) [inline releases/v0.6.0.md:15]: I'd remove the dot at the end of the 2 last bullet points to be consistent with the previous 3 points

File: releases/v0.6.0.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+- AMD SEV-SNP
+
+The following platforms are in development:
+- ARM CCA
+
+## Limitations
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * Image signature validation with AMD SEV-ES is not covered by CI.
```
Comment (ariel-adam) [inline releases/v0.6.0.md:35]: Similar I'd be consistent and use dots at the end of all the bullet points or not at all

**Review by ariel-adam — APPROVED** [review]

File: releases/v0.6.0.md (hunk 4 above)
Comment (fitzthum) [reply releases/v0.6.0.md:15]: This bullet is a sentence where the others are not.

File: releases/v0.6.0.md (hunk 3 above)
Comment (fitzthum) [reply releases/v0.6.0.md:56]: The limitations have been copied from release to release. I will see if I can fix them up a bit.

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @fitzthum

**Review by magowan — APPROVED** [review]

**Pull request merged**